### ISSUE TITLE
Incompatibility with xarray 0.12.0

### ISSUE THREAD

**Issue by MaggieMarvin** [darothen/xbpch#18]
Similar to issue #8: xbpch v0.3.4 seems to be incompatible with xarray v0.12.0

```
>>> import xbpch
Traceback (most recent call last):                                                                      
   File "~/miniconda/lib/python3.7/site-packages/xbpch/__init__.py", line 7       
      from . bpch import BPCHFile                                                                         
   File "~/miniconda/lib/python3.7/site-packages/xbpch/bpch.py", line 11 
      from xarray.core.pycompat import OrderedDict                                                      
ImportError: cannot import name 'OrderedDict' from 'xarray.core.pycompat' 
(~/miniconda/lib/python3.7/site-packages/xarray/core/pycompat.py)
```

**Comment by darothen** [comment]
Thanks for the heads up. I'll try to cut a patch ASAP

**Comment by lizziel** [comment]
This issue is causing problems for new users of https://github.com/geoschem/gcpy/ who are setting up their environments for the first time. What is the timeline for a fix?

**Comment by darothen** [comment]
Pushing commits right now to fix this; will cut a release to v0.3.5 later tonight to include this and a few other small tweaks to improve accessibility.
